static_src/test/unit/components/service_plan.spec.jsx

```diff
@@ -55,3 +55,56 @@ describe('<ServicePlan />', () => {
     expect(props.onAddInstance.calledWith(props.plan.guid)).toBe(true);
   });
 });
+
+describe('<ServicePlan with multi param plan />', () => {
```

Comment:
Nitpick, I think this should be 'with multi param plan', since 'ServicePlan' is already a top level description block.